localnet/poktrolld/config/relayminer_config_full_example.yaml

```diff
@@ -78,18 +83,25 @@ suppliers:
       # Optional.
       headers: {}
 
-    # A list of hosts the HTTP service is offering.
-    # When linked to the proxy, that hostname is going to be used to route the
-    # request to the correct supplier.
-    # That hostname is what the user should stake the supplier for.
-    # Must be unique within a proxy/proxies it is set up on;
-    # in other words, one proxy can't offer the same hostname more than once.
-    # The `name` of the supplier is automatically added to the hosts section
-    # for potential troubleshooting/debugging purposes
+    # A list of hosts the proxy is accepting requests from.
+    # When linked to the proxy, each host is going to be used to lookup the
+    # the Supplier.Service in the Pocket Network.
+    # Each host in the list must match a Supplier.Service.Endpoint that the Supplier
+    # has advertised on-chain when staking for that Service.
+    # The reasons to have multiple hosts for the same supplier service are:
```

Comment:
Same comment as in #288

![Screenshot 2023-12-22 at 4 21 03 PM](https://github.com/pokt-network/poktroll/assets/1892194/6387b3f4-30ce-49d8-9c0a-ac3f4705fdba)